docs/content/en/docs/Getting started/setup-development/index.md

```diff
@@ -1,90 +1,172 @@
 
-Warning: in production you should use your own certificates and not the ones provided by us as this is highly insecure when exposed to the internet and any insecure network. A tutorial to setup PKI infrastructure for MQTT according to [this guide](../../tutorials/pki)
+1. Insert your USB-stick with k3OS into your edge device and boot from it [^boot-usb]
+2. [Install k3OS](/docs/tutorials/install-k3os/). When asked for a cloud-init file, enter this URL and confirm: `https://www.umh.app/development.yaml`. If you are paranoid or want to setup devices for production you could copy the file, modify and host it yourself. [Here is the template](/examples/development.yaml)
 
-1. Go into the folder `/home/rancher/united-manufacturing-hub/deployment/factorycube-edge`
-2. Create a new file called `development_values.yaml` using `touch development_values.yaml`
-3. Copy the following content to that file or use the following example [`development_values.yaml`](/examples/factorycube-server/development_values.yaml). 
-4. **(Only if you did not use example file)** Copy the certificates in `deplotment/factorycube-server/developmentCertificates/pki/` and then `ca.crt`, `issued/TESTING.crt` and `issued/private/TESTING.key` into `development_values.yaml`. Additionally use as `mqttBridgeURL` `ssl://factorycube-server-vernemq-local-service.factorycube-server:8883`. 
-5. Adjust `iprange` to your network IP range
+This process takes around 15 - 20 minutes depending on your internet connection and there will be no further information about the installation status on the output of the device visible (the information on the computer screen).
 
-Example for `development_values.yaml`:
-```yaml
-mqttBridgeURL: "ssl://mqtt.umh.app:8883"
-mqttBridgeTopic: "ia/factoryinsight"
-sensorconnect:
-  iprange: "172.16.1.0/24"
-mqttBridgeCACert: |
-  ENTER CERT HERE
-mqttBridgeCert: |
-  ENTER CERT HERE
-mqttBridgePrivkey: |
-  ENTER CERT HERE
-```
+[^boot-usb]: See also out guide: [How to boot from a USB-stick](/docs/getting-started/understanding-the-technologies/#installing-operating-systems-from-a-usb-stick)
 
-6. Execute `helm install factorycube-edge /home/rancher/united-manufacturing-hub/deployment/factorycube-edge --values "/home/rancher/united-manufacturing-hub/deployment/factorycube-edge/development_values.yaml" --set serialNumber=$(hostname) --kubeconfig /etc/rancher/k3s/k3s.yaml  -n factorycube-edge` (change kubeconfig and serialNumber accordingly) (Please pay attention to the correct path. The path may be /home/rancher/united-manufacturing-hub-main ... or similar)
+#### Getting access to the device
 
-### Install factorycube-server
+To verify whether the installation worked and access [Grafana] (the dashboard) and [Node-RED], we will first enable SSH via password authentification, fetch the login details for [Kubernetes] and then login via [Lens].
```

Comment:
Insert "to":
To verify whether the installation worked and **to** access [Grafana] (the dashboard) and [Node-RED], we will first enable SSH via password authentification, fetch the login details for [Kubernetes] and then login via [Lens].